content/en/blog/kernel-testing.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Kernel testing framework
+date: 2023-09-14
```

Comment:
We released a blog post today, can you wait tomorrow or next Monday?

Reply:
no problem!